guides/schematics.md

```diff
@@ -26,7 +26,7 @@ The Angular Material `ng add` schematic helps you set up an Angular CLI project
 
 - Ensure [project dependencies](./getting-started#step-1-install-angular-material-angular-cdk-and-angular-animations) are placed in `package.json`
 - Enable the [BrowserAnimationsModule](./getting-started#step-2-configure-animations) in your app module
-- Add either a [prebuilt theme](./theming#pre-built-themes) or a [custom theme](./theming#defining-a-custom-theme)
```

Comment:
Do these links no longer exist?

Reply:
The first one does, but the second doesn't. We could just link to our theming guide but Im not sure if theres much value here linking. It seemed self-explanatory that the schematic is adding a theme in general 🤷‍♂️